### DISCUSSION TITLE
xDrip on Galaxy Watch 5 Pro quit working (WearOS 4.0 & UI 5.0).

### DISCUSSION THREAD

**Discussion by tvLors** [NightscoutFoundation/xDrip#3446] [Q&A]
I've been using xDrip+ for years... But lately I have run into issues getting it to work.

Currently, I simply cannot get xDrip to run on my watch (Galaxy 5 Pro; WearOS 4.5 & UI 5.0) - I suspect the problem is related to an OS update, but I really don't know what has gone wrong!

A few days ago, my watch display simply quit updating the xdrip watch face. All other apps worked fine, but xdrip on my watch simply stopped communicating with my phone (Samsung S24 Ultra; Android 14). Out of desperation, I unpaired & then re-paired my watch to my phone. That of course uninstalled everything, forcing me to start over. Getting Wear Installer 2 to work took numerous tries before I had success. But even then, only one icon now shows up in my watch app list whereas 4 existed previously.. When I clicked that icon, it requested a couple permissions and then quit. Now when I click that icon, the screen goes blank for a couple seconds and then exits back to the app menu screen. There is also no watch face to select (on phone or watch); I presume the issue is related to the non-existence of a xdrip watch face??

Any ideas or suggestions? 

FYI, I also had issues on my phone a few weeks earlier. It simply would not connect to my G7 sensor, despite having to previous issues during the 90 days had been using G7s. The sensor still worked fine with my pump. When the sensor expired, I tried a new sensor and it also would not work with xdrip. I eventually gave up and installed the Dexcom G7 app. I then programmed xdrip to use "Companion App" as my data source. That still works OK, and it worked with my watch until a few days ago.

I really miss not having a graph on my watch face.  When I implementing that feature many years ago, my A1C dropped half a point because it made it practical to keep an eye on where things were going, and let me take action sooner.

Thanks for any help and/or suggestions.

**Comment by tvLors** [comment]
Guess what? Posting this question appears to have somehow corrected my problem!

I decided to uninstall the app from my watch and reboot it and reinstall it again - for probably the fifth time! And low & behold, this time all the app icons appeared on the watch and the app works!

I have no idea what was wrong. Earlier I had trouble getting Wear Installer 2 to pair with my phone. I think I tried that about a dozen times before I had success! I also tried using Buglaeger, and while it paired with my phone easily, I don't think it installed xdrip correctly.

So now my only remaining issue is why xdrip has stopped pairing with my G7 sensors, after working fine with those sensors for months. The workaround of using the Dexcom app as a companion source in xDrip works fine, so this issue is no big deal - but it does puzzle me.

**Comment by nekromantik** [comment]
mine still does not load same issue
im running it on Pixel Watch 3 and using Gemnini man to install
